### DISCUSSION TITLE
GitLab general config.

### DISCUSSION THREAD

**Discussion by max-hammer** [renovatebot/renovate#33323] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab

### Please tell us more about your question or problem

Hello together 

I want to have a gitlab repository where I have all renovate configuration files and another gitlab repository where I check for dependencies updates for all repositories using renovate. Is that possible somehow? I would also like to use configuration variables to define which repositories should use which of the renovate configurations, BUT I don't want to have to constantly create renovate.json files. Is that possible?

Kind regards

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by RahulGautamSingh** [comment]
1. See `inheritConfig`, `requireConfig`  in the docs 

2. If you want different config for each repo and store the separate config file for each repo under a different file name then I do not think that is possible easily. 
Maybe, a combination of `extends`, `matchRepositories` in the same config file could work but you will have to try that yourself. We can help debug.

3. I don't think so
    > Re: "another gitlab repository where I check for dependencies updates for all repositories using renovate. "
